lib/nostrum/cache/message_cache/mnesia.ex

```diff
@@ -49,8 +49,7 @@ if Code.ensure_loaded?(:mnesia) do
 
     # allow us to override the table name for testing
     # without accidentally overwriting the production table
-    @table_name @config[:table_name] || :nostrum_messages
-    @record_name @table_name
+    @base_table_name @config[:table_name] || :nostrum_messages
```

Comment:
I know this is mostly a me problem, but because I have it persist the data for this cache across restarts, I'll have to ask for a way to force the table to have a certain name or something that ensures backwards compatibility.